tests/h/models/groups_test.py

```diff
@@ -155,15 +147,57 @@ def test_documents_when_group_has_no_documents(group):
     assert group.documents() == []
 
 
-def test_acl(group, factories):
-    group.pubid = 'testing-pubid'
-    group.creator = factories.User(username='luke', authority='foobar.org')
+class TestGroupACL(object):
+    def test_authority_joinable(self, group):
+        group.joinable_by = JoinableBy.authority
+
+        assert (security.Allow, 'authority:example.com', 'join') in group.__acl__()
```

Comment:
I think these tests should probably be implemented using an instance of `ACLAuthorizationPolicy`, and doing `assert policy.permits(group, ['someprincipal'], 'join')` because asserting that the (allow/deny, principal, permission) triple is in the ACL doesn't really mean much if the first element of the ACL is `security.DENY_ALL`.